pkg/network/controllers/hotplug.go

```diff
@@ -46,19 +46,35 @@ func ApplyDynamicIfaceRequestOnVMI(
 		if shouldBeHotUnplug {
 			vmiIface := vmispec.LookupInterfaceByName(vmiSpecCopy.Domain.Devices.Interfaces, vmIface.Name)
 			vmiIface.State = v1.InterfaceStateAbsent
+		} else {
+			applyInterfaceStateInVMI(vmiSpecCopy, vmIface)
```

Comment:
How about:
if vmi interface exists and its state is different from the vm - set the VMI interface's state to the same value as the VM.